db/migrate/20170531094817_add_organizaton_id_to_conference.rb

```diff
@@ -0,0 +1,5 @@
+class AddOrganizatonIdToConference < ActiveRecord::Migration
+  def change
+    add_column :conferences, :organization_id, :integer
```

Comment:
Since `organization_id` is a required field in conference, we probably want to:
1. First create the field. You can take advantage of the [`add_reference`](http://api.rubyonrails.org/v4.2.5/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-add_reference) method instead of adding an integer. Plus, you probably want to use the `index: true` option, since we'll be filtering conferences by organization.
1. Create a first default organization.
1. Fill out the `organization_id` column on all existing conferences with the default organization's id.
1. Create a [foreign key](http://api.rubyonrails.org/v4.2.5/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-add_foreign_key) so that the conference-organization relation is always valid. 

This should all be done in migrations (not rake tasks), since we are "migrating" the data from the old no-organization format to the new one.